doonut-app-external-api/src/main/kotlin/com/doonutmate/oauth/apple/client/AppleAccessClient.kt

```diff
@@ -0,0 +1,40 @@
+package com.doonutmate.oauth.apple.client
+
+import com.doonutmate.oauth.apple.constants.AppleOauthConstants
+import com.doonutmate.oauth.apple.dto.ApplePublicKeyResponse
+import com.doonutmate.oauth.apple.dto.AppleTokenResponse
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.service.annotation.GetExchange
+import org.springframework.web.service.annotation.HttpExchange
+import org.springframework.web.service.annotation.PostExchange
+
+@HttpExchange(AppleOauthConstants.APPLE_URL)
+interface AppleAccessClient {
+    @GetExchange(AppleOauthConstants.GET_PUBLIC_KEYS)
+    fun getApplePublicKeys(): ApplePublicKeyResponse
+
+    @PostExchange(AppleOauthConstants.CREATE_TOKEN)
+    fun createAppleToken(
+        @RequestParam("client_id") clientId: String,
+        @RequestParam("client_secret") clientSecret: String,
+        @RequestParam("grant_type") grantType: String,
+        @RequestParam("code") code: String,
+    ): AppleTokenResponse
+
+    @PostExchange(AppleOauthConstants.CREATE_TOKEN)
+    fun reissuanceAccessTokens(
```

Comment:
[Kotline Coding converntions](https://kotlinlang.org/docs/coding-conventions.html#choose-good-names)에 맞춰 메서드명은 동사로 짓는게 좋을 것 같습니다.
> The name of a method is usually a verb or a verb phrase saying what the method does: close, readPersons. 
```suggestion
    fun reissueAccessTokens(
```